code/backtracking/n_queen/nqueen.py

```diff
@@ -0,0 +1,48 @@
+
+import json
+global x
+x=[]
+global n
+n=0
+global flag
```

Comment:
how about we initialize all global variable together?
Maybe in here or in main function?

Reply:
Sure thing.